packages/terra-status-view/src/StatusView.jsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Divider from 'terra-divider';
+import styles from './StatusView.scss';
+
+const cx = classNames.bind(styles);
+
+const variants = {
+  NODATA: 'no-data',
+  NOMATCHINGRESULTS: 'no-matching-results',
+  NOTAUTHORIZED: 'not-authorized',
+  ERROR: 'error',
+  CUSTOM: 'custom',
+};
+
+const propTypes = {
```

Comment:
Should there be a prop for a glyph for the custom type?

Reply:
to have the icons be themed, they are provided via the css, therefore, to provide a custom icon you pass in the custom variant and have the css property set.

Reply:
updated to pass in a prop of type node to allow the consumer to provide the glyph